### DISCUSSION TITLE
Watchtower "head request" errors for pgvecto-rs and redis images

### DISCUSSION THREAD

**Discussion by Str1atum** [immich-app/immich#10210] [General]
Seems like the images have some errors and watchtower cannot check them for updates.

See https://github.com/containrrr/watchtower/discussions/1529

![IMG_5209](https://github.com/immich-app/immich/assets/76776216/5dc2444d-e6c1-4155-b066-07d280ba0cb0)

**Comment by bo0tzz** [comment]
These aren't Immich images, so I'm not sure why you're reporting it here ;) That said, you really should not be autoupdating these images with watchtower, that's a recipe for serious breakage. Instead you should pin to specific version tags (which coincidentally would fix this error, as is clearly stated by the error message)
